src/test/java/uk/gov/hmcts/divorce/solicitor/event/SolicitorUpdateAosTest.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.divorce.solicitor.event;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.ccd.sdk.ConfigBuilderImpl;
+import uk.gov.hmcts.divorce.common.model.CaseData;
+import uk.gov.hmcts.divorce.common.model.State;
+import uk.gov.hmcts.divorce.common.model.UserRole;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static uk.gov.hmcts.divorce.solicitor.event.SolicitorUpdateAos.SOLICITOR_UPDATE_AOS;
+
+@ExtendWith(MockitoExtension.class)
+class SolicitorUpdateAosTest {
+    @InjectMocks
+    private SolicitorUpdateAos solicitorUpdateAos;
+
+    @Test
+    void shouldAddConfigurationToConfigBuilder() {
+        final Set<State> stateSet = EnumSet.allOf(State.class);
+        final ConfigBuilderImpl<CaseData, State, UserRole> configBuilder = new ConfigBuilderImpl<>(CaseData.class, stateSet);
+
+        solicitorUpdateAos.configure(configBuilder);
+
+        assertThat(configBuilder.getEvents().get(0).getId(), is(SOLICITOR_UPDATE_AOS));
```

Comment:
worth using assert J as it will be consistent with other tests

Reply:
Replaced with assert J